### PR TITLE
Use `PlaceholderIds` instead of `Placeholders` in `Actions`

### PR DESCRIPTION
**Pull request by Jasopaum**

### CODE REVIEW

File: google/protobuf/empty_pb2.py [hunk 1]
```diff
@@ -19,8 +17,8 @@
   name='google/protobuf/empty.proto',
   package='google.protobuf',
   syntax='proto3',
-  serialized_options=_b('\n\023com.google.protobufB\nEmptyProtoP\001Z\'github.com/golang/protobuf/ptypes/empty\370\001\001\242\002\003GPB\252\002\036Google.Protobuf.WellKnownTypes'),
-  serialized_pb=_b('\n\x1bgoogle/protobuf/empty.proto\x12\x0fgoogle.protobuf\"\x07\n\x05\x45mptyBv\n\x13\x63om.google.protobufB\nEmptyProtoP\x01Z\'github.com/golang/protobuf/ptypes/empty\xf8\x01\x01\xa2\x02\x03GPB\xaa\x02\x1eGoogle.Protobuf.WellKnownTypesb\x06proto3')
+  serialized_options=b'\n\023com.google.protobufB\nEmptyProtoP\001Z\'github.com/golang/protobuf/ptypes/empty\370\001\001\242\002\003GPB\252\002\036Google.Protobuf.WellKnownTypes',
```
Comment (karlhigley) [inline google/protobuf/empty_pb2.py]: Looks like there are still weird formatting changes happening somehow. Maybe related to `buf` versions?

Comment (Jasopaum) [reply google/protobuf/empty_pb2.py]: Hum, I thought we had settled on a similar version. I'm at 0.8.0 right now.
Should we choose a version for everyone to use?

Comment (karlhigley) [reply google/protobuf/empty_pb2.py]: We could probably address this by moving all the stub building into GH actions. We were on 0.7.1 last I knew, but no objections to upgrading.

Comment (Jasopaum) [reply google/protobuf/empty_pb2.py]: I agree on moving the build to GH actions.
The only negative thing I can see about that is that building locally was useful for testing. But we can still do that and only commit .proto files

Comment (karlhigley) [reply google/protobuf/empty_pb2.py]: Even if the other files get committed, I think the action will override them with whatever it thinks the official version should look like. 🤷‍♂

File: protobuf/syft_proto/types/syft/v1/arg.proto [hunk 2]
```diff
@@ -24,6 +23,6 @@ message Arg {
 
         syft_proto.generic.pointers.v1.PointerTensor arg_pointer_tensor = 8;
         syft_proto.execution.v1.Placeholder arg_placeholder = 9;
-        syft_proto.execution.v1.PlaceholderId arg_placeholder_id = 10;
+        syft_proto.execution.v1.PlaceholderId arg_placeholderid = 10;
```
Comment (karlhigley) [inline protobuf/syft_proto/types/syft/v1/arg.proto]: For consistency with the change in `computation_action.proto`:
```suggestion
        syft_proto.execution.v1.PlaceholderId arg_placeholder_id = 10;
```

**Review by karlhigley — APPROVED** [review]

**Pull request merged**